roboflow/core/version.py

```diff
@@ -197,7 +197,7 @@ def download(self, model_format=None, location=None, overwrite: bool = True):
             try:
                 import_module("ultralytics")
                 print_warn_for_wrong_dependencies_versions(
-                    [("ultralytics", "==", "8.0.134")]
+                    [("ultralytics", "==", "8.0.196")]
```

Comment:
if you search for `8.0.134` in this file, we should update all places. I think this is just in the `download` method -- we also need for `deploy` and the relevant print statements

Reply:
👍